libs/utils/decorate.js

```diff
@@ -0,0 +1,146 @@
+//
```

Comment:
File should probably be moved to /libs/utils

Reply:
@chrischrischris  - I don't follow, This is where the file lives already?